src/cloud/release-notes/backward-incompatible-changes.md

```diff
@@ -0,0 +1,112 @@
+   `build_options.ini` file | Use the [`.magento.env.yaml`]({{ site.baseurl }}/cloud/project/magento-env-yaml.html)) file to configure environment variables to manage build and deploy actions across all your environments.<br><br>If you build a Cloud environment that includes the `build_options.ini` file, the build fails.
+
+### CLI command changes
+
+The following table summarizes CLI command changes in {{ site.data.var.ct }} v2002.1.0 that might require you to update commands or scripts.
+
+ Command| Replacement
+ -------- |-------
+`m2-ece-build` | `vendor/bin/ece-tools build`
+`m2-ece-deploy` | `vendor/bin/ece-tools deploy`
+`m2-ece-scd-dump` | `vendor/bin/ece-tools config:dump`
+`vendor/bin/ece-tools patch` | `vendor/bin/ece-patches apply`
+`vendor/bin/ece-tools docker:build` | `vendor/bin/ece-docker build:compose`
+`vendor/bin/ece-tools docker:config:convert` | `vendor/bin/ece-docker  image:generate:php`
+
+In previous {{ site.data.var.ct }} releases, you could use the `m2-ece-build` and `m2-ece-deploy` commands to configure deployment hooks in the `.magento.app.yaml` file. When you update to v2002.1.0, check the `hooks` configuration in the `.magento.app.yaml` file for the obsolete commands, and replace them if needed.
+
+## Magento Cloud Patches changes
+
+-  **Remove downloaded patches**–The `{{site.data.var.mcp}}` package bundles all patches available from the [Magento Technical resources](https://magento.com/tech-resources/download) page and applies them automatically when you deploy to the Cloud.  To prevent patch conflicts after upgrading to {{site.data.var.ct}} 2002.1.0 or later, remove any Magento-supplied patches that you downloaded and added to your project manually.
+
+-  **Updating the apply patches command**–We moved the command for applying patches to Magento Cloud from the `vendor/bin/ece-tools` directory to the `vendor/bin/ece-patches` directory. If you use this command to apply patches manually, use the new path.
+
+   > Manually apply patches
+
+   ```bash
+   php ./vendor/bin/ece-patches apply
+   ```
+
+## Magento Cloud Docker changes
+
+-  **The minimum PHP version requirement is now PHP 7.1**–If your Magento Cloud Docker host is running an earlier version, upgrade to PHP v7.1 or later.
+
+-  **Magento Cloud Docker command changes**–
+
+   -  **Updating Magento Cloud Docker commands for Docker build operations**–We moved the Magento Cloud Docker commands from the `vendor/bin/ece-tools` directory to the `vendor/bin/ece-docker` directory. Update your scripts and commands to use the new path.
+
+      After upgrading to the {{site.data.var.ct}} 2002.1.0 or {{site.data.var.mcd}} 1.0.0, use the following command to view available `ece-docker` commands.
+
+      ```bash
+      php ./vendor/bin/ece-docker list
+      ```
+
+   -  **Updating the Magento Cloud docker-compose commands**–We changed the name of the `./bin/docker` directory to `./bin/magento-docker`.  Update your scripts and commands to use the new path.
```

Comment:
The directory didn't change, but the path did. We could say it was renamed instead.
```suggestion
   -  **Updating the Magento Cloud docker-compose commands**–We renamed `./bin/docker` to `./bin/magento-docker`.  Update your scripts and commands to use the new path.
```